src/final2/subtests/LangtonSubtest.java

```diff
@@ -39,4 +86,29 @@ protected String consoleMessage(String[] commands, String[] commandLineArguments
 				+ joinOnePerLine(commands) + "\n\nbut got unexpected output:\n";
 		return result;
 	}
+	
+	/**
+	 * Checks if the given expected pitch matches the program output.
+	 * @param input the input file
+	 * @param pExpectedPitch the lines of the expected pitch
+	 * @return the run which checks the pitch
+	 */
+	protected Run checkPitch(String[] input, String[] pExpectedPitch) {
```

Comment:
yes, you are right ;)

2015-03-21 20:54 GMT+01:00 pcworld <notifications@github.com>:

> In src/final2/subtests/LangtonSubtest.java
> <https://github.com/jGleitz/JUnit-KIT/pull/110#discussion_r26896253>:
>
> > @@ -39,4 +86,29 @@ protected String consoleMessage(String[] commands, String[] commandLineArguments
> >  				+ joinOnePerLine(commands) + "\n\nbut got unexpected output:\n";
> >  		return result;
> >  	}
> > +	
> > +	/**
> > +	 * Checks if the given expected pitch matches the program output.
> > +	 * @param input the input file
> > +	 * @param pExpectedPitch the lines of the expected pitch
> > +	 * @return the run which checks the pitch
> > +	 */
> > +	protected Run checkPitch(String[] input, String[] pExpectedPitch) {
>
> Seems like input should be removed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jGleitz/JUnit-KIT/pull/110/files#r26896253>.
>